rfc/index.md

```diff
@@ -119,24 +122,24 @@ merged into the RFC repository as a markdown file. At that point the RFC is
     request, and leave a comment on the pull request explaining your changes.
     Specifically, do not squash or rebase commits after they are visible on the
     pull request.
-  - At some point, a member of the subteam will propose a "motion for final
+  - At some point, a member of the sub-team will propose a "motion for final
     comment period" (FCP), along with a *disposition* for the RFC (merge, close,
     or postpone).
     <!-- alex ignore clearly -->
   - This step is taken when enough of the tradeoffs have been discussed that
-    the subteam is in a position to make a decision. That does not require
+    the sub-team is in a position to make a decision. That does not require
     consensus amongst all participants in the RFC thread (which is usually
     impossible). However, the argument supporting the disposition on the RFC
     needs to have already been clearly articulated, and there should not be a
-    strong consensus *against* that position outside of the subteam. Subteam
+    strong consensus *against* that position outside of the sub-team. Sub-team
     members use their best judgment in taking this step, and the FCP itself
     ensures there is ample time and notification for stakeholders to push back
     if it is made prematurely.
     - For RFCs with lengthy discussion, the motion to FCP is usually preceded by
       a *summary comment* trying to lay out the current state of the discussion
       and major tradeoffs/points of disagreement.
-    - Before actually entering FCP, *all* members of the subteam must sign off;
-    this is often the point at which many subteam members first review the RFC
+    - Before actually entering FCP, *all* members of the sub-team must sign off;
```

Comment:
Do we want to sign off from everyone? This makes things like vacations, illness, a process problem. Perhaps we can use `a majority and no objections`?

Reply:
This is consistent with other sub-teams - We probably want to reconsider that though ... so we can all have vacations!

Reply:
ja I was thinking about this in general (for all the teams) I'll drag it into a ticket